### PR TITLE
fixed highlight and made it dynamic based on url

### LINKED ISSUE #5: Header Sign-in functionality
Update header to match caesars.com/careers/ site

### PR DESCRIPTION
**Pull request by amol-anand**
Fix #5 

## 🔗 Test URLs:

- Before: https://main--caesars-careers--hlxsites.hlx.page/sms-terms-and-conditions
- After: https://issue-5-header-fixes--caesars-careers--hlxsites.hlx.page/sms-terms-and-conditions

## 📝 Description:

Highlight menu items based on current URL and fixed the animation on the highlights

### CODE REVIEW

File: careers/blocks/header/header.css [hunk 1]
```diff
@@ -605,14 +606,22 @@ header nav .nav-tools p:first-of-type a:hover {
...
-    margin: auto 0;
+    left: 0;
+    bottom: -10px;
+    transform: scaleX(0);
+    transition: transform .15s cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  header nav .nav-sections .local-nav ul li a.active::after,
+  header nav .nav-sections .local-nav ul li a:hover::after {
+    transform: scaleX(1);
```
Comment (tmathern) [inline careers/blocks/header/header.css:624]: Question: Why is this needed? What does it do?

**Review by tmathern — APPROVED** [review]
Code LGTM - Do you want to look at the logo misalignment too in this PR?

**Review by tmathern — APPROVED** [review]
Code LGTM - Do you want to look at the logo misalignment too in this PR?

File: careers/blocks/header/header.css (hunk 1 above)
Comment (amol-anand) [reply careers/blocks/header/header.css:624]: It basically says the we moved from scaleX(0) to scaleX(1) on hover - which means that white block under the nav items will grow from 0 to 1 horizontally (X axis) which gives that cool animation effect

**Comment by amol-anand** [comment]
> Code LGTM - Do you want to look at the logo misalignment too in this PR?

I fixed that in the CSS.. which misalignment you talking about?

**Comment by amol-anand** [comment]
If you look at https://www.caesars.com/careers it's the same. the logo is not misaligned, it is intentionally bigger than the rest of the nav and stands out and overflows down the nav a bit overlaying the hero.
